### ISSUE TITLE
No field with type int exists in class PacketPlayOutEntityDestroy

### ISSUE THREAD

**Issue by MatsG23** [josemmo/yamipa#4]
Hello,

after the upgrade to 1.17.1 this error occurs:
```
[15:22:42 ERROR]: Could not pass event PlayerTeleportEvent to YamipaPlugin v1.1.0-SNAPSHOT
com.comphenix.protocol.reflect.FieldAccessException: No field with type int exists in class PacketPlayOutEntityDestroy.
        at com.comphenix.protocol.reflect.StructureModifier.writeInternal(StructureModifier.java:365) ~[?:?]
        at com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:345) ~[?:?]
        at io.josemmo.bukkit.plugin.packets.DestroyEntityPacket.setId(DestroyEntityPacket.java:17) ~[?:?]
        at io.josemmo.bukkit.plugin.renderer.FakeItemFrame.destroy(FakeItemFrame.java:145) ~[?:?]
        at io.josemmo.bukkit.plugin.renderer.FakeImage.destroy(FakeImage.java:290) ~[?:?]
        at io.josemmo.bukkit.plugin.renderer.WorldArea.unload(WorldArea.java:110) ~[?:?]
        at io.josemmo.bukkit.plugin.renderer.ImageRenderer.onPlayerLocationChange(ImageRenderer.java:343) ~[?:?]
        at io.josemmo.bukkit.plugin.renderer.ImageRenderer.onPlayerTeleport(ImageRenderer.java:388) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor192.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.17.1.jar:git-Paper-90]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.17.1.jar:git-Paper-90]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.17.1.jar:git-Paper-90]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.17.1.jar:git-Paper-90]
        at org.bukkit.craftbukkit.v1_17_R1.entity.CraftPlayer.teleport(CraftPlayer.java:954) ~[patched_1.17.1.jar:git-Paper-90]
        at org.bukkit.craftbukkit.v1_17_R1.entity.CraftEntity.teleport(CraftEntity.java:544) ~[patched_1.17.1.jar:git-Paper-90]
        at com.onarandombox.MultiverseCore.utils.SimpleSafeTTeleporter.safelyTeleport(SimpleSafeTTeleporter.java:216) ~[?:?]
        at com.onarandombox.MultiversePortals.listeners.PlayerListenerHelper.performTeleport(PlayerListenerHelper.java:44) ~[?:?]
        at com.onarandombox.MultiversePortals.listeners.MVPPlayerMoveListener.playerMove(MVPPlayerMoveListener.java:167) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor233.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.17.1.jar:git-Paper-90]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.17.1.jar:git-Paper-90]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.17.1.jar:git-Paper-90]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleMovePlayer(ServerGamePacketListenerImpl.java:1420) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundMovePlayerPacket.handle(ServerboundMovePlayerPacket.java:114) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundMovePlayerPacket$Pos.handle(ServerboundMovePlayerPacket.java:42) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:36) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1348) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.server.MinecraftServer.shouldRun(MinecraftServer.java:190) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1327) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1320) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1281) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1192) ~[patched_1.17.1.jar:git-Paper-90]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[patched_1.17.1.jar:git-Paper-90]
        at java.lang.Thread.run(Thread.java:831) [?:?]
```
I have already installed the latest 4.7.0 version of ProtocolLib that was announced by the update checker.

Best Regards
Mats

**Comment by MatsG23** [comment]
Yikes!
Just seeing that you already added a commit including a fix for that...

**Comment by josemmo** [comment]
Yup, just released [v1.1.1](https://github.com/josemmo/yamipa/releases/tag/v1.1.1) which hopefully should fix this bug.

Thanks for letting me know, tho! :)
